qemu/tests/cfg/block_performance_test.cfg

```diff
@@ -0,0 +1,35 @@
+- block_performance_test:
+    type = block_performance_test
+    only virtio_blk, virtio_scsi
+    images += " stg0 stg1"
+    force_create_image_stg0 = yes
+    force_remove_image_stg0 = yes
+    image_size_stg0 = 10G
+    image_name_stg0 = stg0
+    force_create_image_stg1 = yes
+    force_remove_image_stg1 = yes
+    image_size_stg1 = 11G
+    image_name_stg1 = stg1
+    kill_vm = yes
+    dd_cmd_timeout = 180
+    Linux:
+        dd_cmd = "time dd if=/dev/zero of=/dev/%s bs=1M count=5000 oflag=direct"
+    Windows:
+        dd_cmd = 'WIN_UTILS:\coreutils\DummyCMD.exe %s 1024000000 1'
+    variants:
+        - queue_size_test:
+            virtio_blk:
+                blk_extra_params_stg0 = "queue-size=128"
+                blk_extra_params_stg1 = "queue-size=1024"
+            virtio_scsi:
+                drive_bus_stg0 = 1
+                drive_bus_stg1 = 2
+                bus_extra_params_stg0 = "virtqueue_size=128"
+                bus_extra_params_stg1 = "virtqueue_size=1024"
+        - multi_queue_test:
+            vcpu_maxcpus = 8
+            virtio_blk:
+                blk_extra_params_stg1 = "num-queues=${vcpu_maxcpus}"
+            virtio_scsi:
+                drive_bus_stg1 = 1
+                bus_extra_params_stg1 = "num_queues=${vcpu_maxcpus}"
```

Comment:
num_queues=<uint32>    -  (default: 4294967295)
Sincs the default value of num_queues in qemu is not clearly specified, it is recommended that `num_queues=1` in bus_extra_params_stg0  give a clearly smaller value for easy comparison.

Reply:
Ok, it makes sense, thank you!